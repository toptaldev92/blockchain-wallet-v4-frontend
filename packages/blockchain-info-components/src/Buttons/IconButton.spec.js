import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

import IconButton from './IconButton'

describe('IconButton component', () => {
  it('default renders correctly', () => {
    const component = shallow(<IconButton name="test" />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
