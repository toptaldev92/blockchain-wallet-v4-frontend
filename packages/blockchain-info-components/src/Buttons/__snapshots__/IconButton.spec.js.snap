// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton component default renders correctly 1`] = `
<Styled(Button)>
  <Icon
    color="gray-6"
    cursor={false}
    name="test"
    size="16px"
    weight={400}
  />
</Styled(Button)>
`;
